docs/CUSTOM_RULES_101.md

```diff
@@ -40,10 +40,10 @@ This project already contains custom rules. Our goal will be to add an extra rul
 
 A custom plugin is a Maven project, and before diving into code, it is important to notice a few relevant lines related to the configuration of your soon-to-be-released custom plugin.
 
-In the code snippet below, note the plugin API version (`<sonar.version>`) provided through the properties. It relates to the minimum version of SonarQube your plugin will support, and is generally aligned to your company's SonarQube instance. In this template, we rely on the version **7.7** (LTS version is **6.7**, but compatibility to latest release is guaranteed when packaging the plugin).
+In the code snippet below, note the plugin API version (`<sonar.version>`) provided through the properties. It relates to the minimum version of SonarQube your plugin will support, and is generally aligned to your company's SonarQube instance. In this template, we rely on the version **8.2.0.32929** (LTS version is **7.9**, but compatibility to latest release is guaranteed when packaging the plugin).
 Note that latest released versions of the Java Analyzer are always compatible with the current LTS version of SonarQube.
 
-The property `<java.plugin.version>` is the minimum version of the Java Analyzer that will be required to run your custom plugin in your SonarQube instance. Consequently, as we will rely on version **5.12.1.17771** of the Java plugin, the SonarQube instance which will use the custom plugin will also need version **5.12.1.17771** of the Java Plugin as well.
+The property `<java.plugin.version>` is the minimum version of the Java Analyzer that will be required to run your custom plugin in your SonarQube instance. Consequently, as we will rely on version **6.1.0.20866** of the Java plugin, the SonarQube instance which will use the custom plugin will also need version **6.1.0.20866** of the Java Plugin as well.
```

Comment:
now that release 6.2 has been done, maybe we should use it! It should be almost transparent!